pyproject.toml

```diff
@@ -21,6 +21,7 @@ termcolor = "^2.3.0"
 typing-extensions = "^4.6.3"
 loguru = "^0.7.0"
 pydantic = "^1.10.9"
+unstructured = {version="^0.9.0", extras=["pdf"]}
```

Comment:
Add `Chroma` to `pyproject.toml` please!

Reply:
this is done